ledfx/effects/concentric.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import voluptuous as vol
+from PIL import Image
+
+from ledfx.effects import blur_pixels
+from ledfx.effects.audio import AudioReactiveEffect
+from ledfx.effects.gradient import GradientEffect
+from ledfx.effects.twod import Twod
+
+
+class Concentric(Twod, GradientEffect):
+    """
+    A 2D effect that renders concentric circles expanding from the center
+    on the beat. The colors of the circles are determined by a gradient.
+    """
+
+    NAME = "Concentric"
+    CATEGORY = "2D"
+
+    HIDDEN_KEYS = [
+        "background_brightness",
+        "mirror",
+        "flip",
+        "dump",
+        "gradient_roll",
+        "rotate",
+    ]
+
+    CONFIG_SCHEMA = vol.Schema(
+        {
+            vol.Optional(
+                "frequency_range",
+                description="Frequency range for beat detection",
+                default="Lows (beat+bass)",
+            ): vol.In(list(AudioReactiveEffect.POWER_FUNCS_MAPPING.keys())),
+            vol.Optional(
```

Comment:
Blur is a standard keyword, don't redefine it. If you want a different blur. Then use a different name, force normal Blur to zero and hide it. Careful you don't get double Blur!